code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -1,3 +1,13 @@
+// Defines for research points //
+
+#define SLIME_TIER_0 100
+#define SLIME_TIER_1 500
+#define SLIME_TIER_2 1000
+#define SLIME_TIER_3 1500
+#define SLIME_TIER_4 2000
+#define SLIME_TIER_5 2500
+#define SLIME_TIER_RAINBOW 5000
+
```

Comment:
Move these to the _Defines folder and make them something more descriptive regarding their purpose.